Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.16.5 as builder
+FROM golang:1.18 as builder
```

Comment:
this is no longer pinning to a specific patch version and always pulls in the latest on the 1.18.x series, and I think this matches the GitHub Actions behavior - this means we'll get bug fixes and security updates